### DISCUSSION TITLE
Ring the Bell 🔔 School is now in Session! [Education Check-in]

### DISCUSSION THREAD

**Discussion by samus-aran** [community/community#52831] [GitHub Education]
Welcome to our first Education check-in! Today’s curriculum is _chalk_ full of superstar students, latest goodies added to our Student Developer Pack, and new teacher’s tools 📏 🍎

## Star awards to these Education Community citizens ⭐
- @shahidulislam-cse for sharing a number of [fantastic resources ](https://github.com/orgs/community/discussions/45477#discussioncomment-4793922)on how to get started with Python 🐍 and Dart 🎯 
- Location, location, location. Thanks to @xixilaladsds and @badbanii for providing guidance to [enable a student’s browser access](https://github.com/orgs/community/discussions/42246) needed for GitHub Global Campus verification 🌍
- @Ntshangase you super star! 🌠 Offering advice to students on how to reevaluate their applications as well as helping out in our other community categories! What a great contributor to the community 💟 👏 

## New Partners added to GitHub Student Developer Pack 🧑‍🎓

**Pens and paper down!** ✏️ We’ve excited to announce that we’ve welcomed three new partners to the Student Developer Pack! Learn more 👇

- [Deepnote](https://deepnote.com/) is a new kind of data notebook built for collaboration - Jupyter compatible, works magically in the cloud, and sharing is as easy as sending a link allowing for instantaneous collaboration through shared notebooks and workspaces. The pack offer includes free use of their powerful Deepnote Team plan while you are a student, which features unlimited team members & projects, 30-day version history, premium integrations and more.
- [AlgoExpert](https://www.algoexpert.io/product) is the ultimate resource for coding interviews that includes video explanations, mock interview questions, whiteboard help in multiple languages and more. Gain free access to 20 coding interview questions on AlgoExpert as well as a 10% discount on all AlgoExpert products.
- [Scrimba](https://scrimba.com/) is an interactive learning platform for frontend developers. Level up your coding skills with interactive courses, projects, and challenges. Learn JavaScript, CSS, React, Python, and more. Enjoy 1 month of free access to Full access to Scrimba’s Pro courses, projects, and coding challenges, which includes 40+ courses.

**Whao, full marks!** 💯 🤩
Comment below which of these new offers you’re excited to try out 💬 

## Ready for a new Experience?

Check out the newest experience addition to the GitHub Global Campus and the Student Developer Pack—[Career Readiness](https://education.github.com/experiences/career_readiness)! Get job-search ready today.

The Career Readiness Experience takes you on a 4 module journey through different exercises designed to help you organize your skills and talents to launch a job, internship or [Octernship](https://education.github.com/students/octernships) search. We'll cover everything from identifying your career goals and building your personal brand to networking, preparing your application materials, and acing your interviews.

## Teacher’s Corner 🍎
[Introducing the GitHub Classroom CLI extension for the GitHub CLI](https://github.blog/changelog/2023-03-23-introducing-the-github-classroom-cli-extension-for-the-github-cli/)

We're thrilled to introduce the GitHub Classroom CLI extension for the GitHub CLI, designed to simplify the lives of teachers everywhere. With this powerful new tooling, you can create your own personalized workflows, as well as streamline any custom solutions you've already built.

We want to thank all the teachers that shared their experiences and identified this as a need. You can report issues or request features on [our public repository](https://github.com/github/gh-classroom), where we look forward to open sourcing the code in the coming weeks. 

Remember to share what you build with the GitHub Classroom CLI in our [Global Campus for Teachers Discussions forum](https://github.com/community/Global-Campus-Teachers/discussions)!

**Ding Ding!** 🔔 That’s all for today’s class! Leave a reaction or a comment and share what’s your favorite offer available on the GitHub Student Developer Pack 👍🎒

In order to keep this thread on topic any comments asking for assistance on verifying their application will be marked as off-topic and hidden. We'd appreciate it if you created a new discussion for these types of questions 🆕 ⭐

**Comment by hadesgeek** [comment]
ありがとうございます

**Comment by tully-8888** [comment]
Amazing, good luck to everyone!

**Comment by itsdarkexe7** [comment]
Hello there, I am applying for student pack and I am applying as my school ID card but when upload the image I get an error, if anyone can help?
